test/unit/base.test.js

```diff
@@ -37,6 +40,28 @@ describe('CONVERT FUNCTION TESTS ', function() {
         expect(conversionResult.output[0].data).to.have.property('info');
         expect(conversionResult.output[0].data).to.have.property('item');
 
+        done();
+      });
+    });
+    it('[Github: 102]- Should generate collection info with only contact info' +
+      specPath2, function(done) {
+      Converter.convert({ type: 'file', data: specPath2 }, { schemaFaker: true }, (err, conversionResult) => {
+        let description;
+        description = conversionResult.output[0].data.info.description;
+        expect(description.content).to
+          .equal('Contact Support:\n Name: API Support\n Email: support@example.com');
+
+        done();
+      });
+    });
+    it('[Github: 102]- Should generate collection info with only description' +
```

Comment:
why was this (and line 62) split across 2 lines? was the 80-character limit hit?

Reply:
Yes.